app/src/main/java/com/dessalines/thumbkey/keyboards/ESMessageEase.kt

```diff
@@ -692,40 +694,349 @@ val KB_ES_MESSAGEEASE_SHIFTED =
         ),
     )
 
-val NUM_KEY_2 = NUMERIC_KEYBOARD_MESSAGEASE.arr[0][1]
-val NUM_KEY_3 = NUMERIC_KEYBOARD_MESSAGEASE.arr[0][2]
-val KB_ES_MESSAGEEASE_NUM =
-    NUMERIC_KEYBOARD_MESSAGEASE.copy(
+val KB_ES_MESSAGEEASE_NUMERIC =
```

Comment:
Shouldn't these numeric ones go into a common folder? Since people are making messageease variants that might use them?

IE en_messageease and en_symbols_messageease

Reply:
`KB_EN_MESSAGEEASE_NUMERIC` and `KB_ES_MESSAGEEASE_NUMERIC` can still be used by other keyboards even if it's defined, I know because I had to fix a reference in the "english symbols messageease" keyboard.

But if you're saying it because of discoverability, I guess they could be in separate files. I just didn't want to add even more files to the already very long list in that folder

Reply:
Yes please put them in other files. If multiple keyboards are going to use them, they should live either in common or their own files.